src/main/java/team/asd/service/PersonService.java

```diff
@@ -0,0 +1,60 @@
+package team.asd.service;
+
+import lombok.NonNull;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import team.asd.entities.IsPerson;
+
+import java.util.Collections;
+import java.util.IntSummaryStatistics;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class PersonService implements IsPersonService {
+	@Override
+	public @NonNull List<IsPerson> collectPersonsWithNameStartsWith(List<IsPerson> personList, String prefix) {
+		if (CollectionUtils.isEmpty(personList))
```

Comment:
Need to wrap with `{}` even single line.
E.g.: 
```
if (CollectionUtils.isEmpty(personList)) {
			return Collections.emptyList();
}
```

Reply:
done